appeals/web/src/server/appeals/appeal-details/interested-party-comments/redact/redact.mapper.js

```diff
@@ -17,7 +17,33 @@ export const redactInterestedPartyCommentPage = (appealDetails, comment, session
 	const shortReference = appealShortReference(appealDetails.appealReference);
 
 	/** @type {PageComponent[]} */
-	const pageComponents = [subtitle, instructionsList, form(comment, session)];
+	const pageComponents = [
+		wrapComponents(
+			[
+				{
```

Comment:
A page component helper for `html` component was just added, you may want to use that here: https://github.com/Planning-Inspectorate/appeals-back-office/blob/main/appeals/web/src/server/lib/mappers/components/page-components/html.js